source/portable/mbedtls/core_pkcs11_mbedtls.c

```diff
@@ -4741,6 +4929,40 @@ CK_DECLARE_FUNCTION( CK_RV, C_Verify )( CK_SESSION_HANDLE hSession,
                     }
                 }
             }
+            else if( pxSessionObj->xOperationVerifyMechanism == CKM_AES_CMAC )
+            {
```

Comment:
At the end of this block, no matter the result, can you set the CMAC object handle to invalid